xsd/netex_part_1/part1_tacticalPlanning/netex_stopAssignment_version.xsd

```diff
@@ -177,7 +177,7 @@ Rail transport, Roads and Road transport
 	<!-- ======================================================================= -->
 	<xsd:element name="PassengerStopAssignment" abstract="false" substitutionGroup="StopAssignment">
 		<xsd:annotation>
-			<xsd:documentation>Assignment of a SCHEDULED STOP POINT to a STOP PLACE and QUAY, etc.. For associations to ZONE see FlexibleStopAssignment.</xsd:documentation>
+			<xsd:documentation>Assignment of a SCHEDULED STOP POINT to a STOP PLACE and QUAY, etc.. For associations to ZONE see FlexibleStopAssignment. Usually it is expected to have one single PASSENGER STOP ASSIGNMENT for each combination of STOP PLACE/QUAY/BOARDING POSITION assigned to a SCHEDULED STOP POINT. However, this can't be guaranteed as in some cases operational requirements need multiple combinations of multipe SCHEDULED STOP POINT to a combination in the physical world or even a single SCHEDULED STOP POINT potentially to be connected to multiple combinations in the real world. Also, for some operators the link between the timetable world (SCHEDULED STOP POINT) and the physical may not be provided. This should be avoided, as the link then needs to be done by matching which is not 100% acurate.</xsd:documentation>
```

Comment:
```suggestion
			<xsd:documentation>Assignment of a SCHEDULED STOP POINT to a STOP PLACE and QUAY, etc.. For associations to ZONE see FlexibleStopAssignment. Usually it is expected to have one single PASSENGER STOP ASSIGNMENT for each combination of STOP PLACE/QUAY/BOARDING POSITION assigned to a SCHEDULED STOP POINT. However, this can't be guaranteed as in some cases operational requirements need multiple combinations of multipe SCHEDULED STOP POINT to a combination in the physical world or even a single SCHEDULED STOP POINT potentially to be connected to multiple combinations in the real world. Also, for some operators the link between the timetable world (SCHEDULED STOP POINT) and the physical may not be provided. This should be avoided, as the link then needs to be done by matching which is not 100% accurate.</xsd:documentation>
```